lib/internal/Magento/Framework/Mview/ProcessorInterface.php

```diff
@@ -7,7 +7,7 @@
 
 /**
  * Interface \Magento\Framework\Mview\ProcessorInterface
```

Comment:
Maybe quick description what this interface does, instead of writing the namespace of the ActionInterface.php?